proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -82,7 +81,7 @@ message GenericBeaconBlock {
         BlindedBeaconBlockDeneb blinded_deneb  = 8;
     }
     bool is_blinded = 100;
-    uint64 payload_value = 101;
+    string payload_value = 101;
```

Comment:
same question here, if it should be a string or uint64, we'd need to add checks if it's a uint then right?

Reply:
It can't be a `uint64` because the number is too large. When converting from string to `big.Int`, you use https://pkg.go.dev/math/big#Int.SetString